dc_kinesis_clickhouse_transfer/README.md

```diff
@@ -0,0 +1,236 @@
+Here are the grammar edits for the provided Markdown:
+
+## Clickstream App
+
+Clickstream Demo app for real-time analytics powered by Double.Cloud.
+
+This example creates a simple clickstream infrastructure:
+
+1. Kinesis stream buffer
+2. Clickhouse as click storage
+3. Transfer as click delivery mechanism
+
+**Baseline Architecture**
+
+![architecture.png](./assets/architecture.png)
+
+
+Clusters would be visible in UI:
+
+**Clickhouse:**
+
+![create_clickhouse.png](./assets/create_clickhouse.png)
+
+Once we have clusters set up, we can start building a clickstream solution from it.
+
+**Transfer:**
+
+![create_transfer.png](./assets/create_transfer.png)
+
+### Upload sample data
+
+For testing purposes, you can use the AwsCli to insert data into the kinesis stream. This only requires having AwsCli installed. In addition to the previous step, AWS profile is required to be authenticated and has permissions to create Kinesis stream and insert in kinesis.
+For production use cases, you should consider setting up networking access via [this](https://double.cloud/docs/en/vpc/connect-dc-to-aws) guide.
+
+```shell
+aws kinesis put-record \ 
+--stream-name TERRAFORM_TEST_STREAM_1 \ 
+--partition-key 1 \
+--cli-binary-format raw-in-base64-out \
+--data '{"user_id":"user5", "score":300}' \
```

Comment:
this data is not matched with transfer schema from transfer bellow, that's why this data goes to unparsed, it's better to put data that matched.